### ISSUE TITLE
Support Tor/SOCKS5

### ISSUE THREAD

**Issue by steelman** [SecUSo/privacy-friendly-weather#17]
Please support connections via Tor/SOCKS5 for better privact

**Comment by Yonjuni** [comment]
By the usage of [Orbot](https://guardianproject.info/apps/orbot/) the app' request will be rooted through the Tor network.

**Comment by steelman** [comment]
Does the "No integration planned" mean you don't think this feature is useful?

**Comment by Yonjuni** [comment]
@steelman No, it just means that we do not plan to integrate it for several reasons. But we would like to keep the issue open in case some-one wishes.

**Comment by merge** [comment]
While it is possible for a rooted android Orbot to route traffic through tor, or use Orbot's VPN mode, I think it is a benefit for an application to offer _network proxy settings_.

For any application that does networking. It won't require root for users, and offers more safety. Users then can be sure that **no** connection is being established in case Orbot is killed or whatever. Network proxy settings should be quite lightweight to implement. Many applications offer it.

**Comment by Yonjuni** [comment]
@merge Could you give an example of an app that implements the features you described?

**Comment by steelman** [comment]
[OpenKeychain](https://github.com/open-keychain/open-keychain)

**Comment by merge** [comment]
https://github.com/osmandapp/Osmand

**Comment by Yonjuni** [comment]
Thanks :)
